source/includes/fundamentals/code-snippets/CompoundOperatorsIndividualExamples.java

```diff
@@ -6,6 +6,9 @@
 import com.mongodb.client.MongoDatabase;
 import com.mongodb.client.model.*;
 import com.mongodb.client.result.InsertOneResult;
+
+import docs.builders.Filters;
```

Comment:
Question:
Is this import necessary? From a quick scan, I can't tell which file is in the docs.builders package. Please let me know if there's any reason not to delete it.